core/.bazelrc

```diff
@@ -0,0 +1,35 @@
+build --host_platform=@rules_nixpkgs_core//platforms:host
+
+# NOTE: all of the following is copy-pasted from `/.bazelrc` and necessary to
+# make this repository work compatibly in isolation.
```

Comment:
Perhaps the common parts could be placed in a `.bazelrc.common` and then `import`ed by both `core/.bazelrc` and `/.bazelrc`?

Reply:
Even easier, just add another symlink.